testsuite/python/interactions_non-bonded.py

```diff
@@ -588,40 +588,167 @@ def test_gaussian(self):
 
         self.system.non_bonded_inter[0, 0].gaussian.set_params(eps=0.)
 
+    # Test the Gay-Berne potential and the resulting force and torque
     @utx.skipIfMissingFeatures("GAY_BERNE")
     def test_gb(self):
+
+        # helper function definitions
+        def calc_partial_derivatives(func, x0, dx=1.0e-7):    
```

Comment:
aka gradient

Reply:
Ok, did you rename it and update the docstring?

Reply:
Yes: 377b216

Reply:
ty